buildspec/windowsTests.yml

```diff
@@ -1,6 +1,10 @@
 version: 0.2
 
+env:
+  variables:
+    CI: true
+
```

Comment:
Do we need to explicitly provide `openjdk8` install as well in the build spec?

Reply:
No, only the Ubuntu 2.0 image uses that